packages/login-ui/components/NetworkCreateModalForm/validate.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import ErrorList from 'platform/ErrorList'
```

Comment:
Please use @chronobank/core-dependencies.
Also please check all your PR for 'platform' usage.